RAT/inputs.py

```diff
@@ -151,6 +151,38 @@ def make_problem(project: RAT.Project) -> ProblemDefinition:
     return problem
 
 
+def make_resample(project: RAT.Project) -> list[int]:
+    """Constructs the "resample" field of the problem input required for the compiled RAT code.
+
+    Parameters
+    ----------
+    project : RAT.Project
+        The project model, which defines the physical system under study.
+
+    Returns
+    -------
+     : list[int]
+        The "resample" field of the problem input used in the compiled RAT code.
+    """
+    return [contrast.resample for contrast in project.contrasts]
```

Comment:
I am not a fan of one line functions are these reused anywhere?

Reply:
Found it,  reused in plotting

Reply:
Yeah, I'm not a fan either, but there's madness to the method this time.